mongodb/collection.go

```diff
@@ -148,7 +148,11 @@ func (c *Collection) DeleteById(ctx context.Context, id interface{}) (*Collectio
 	return c.Delete(ctx, selector)
 }
 
-// Aggreate starts a pipeline operation
+// Aggregate starts a pipeline operation
 func (c *Collection) Aggregate(pipeline interface{}) *Cursor {
 	return newCursor(newAggregateCursor(c.collection, pipeline, options.Aggregate()))
 }
+
+func (c *Collection) GetMongoCollection() *mongo.Collection {
```

Comment:
Well we expose the underlying Mongo Collection in our MongoConnection.GetMongoCollection(), which I thought was a 'catch-all' solution of allowing a kick-down to underlying Mongo for functionality we had not wrapped. The only issue with that is that it only allows access to the collection that is wrapped by the Connection. So a collection that is accessed by MongoCollection.C() does not have a facility to access its underlying Mongo Collection. My addition adds this ability.
If the original idea was to have this 'catch-all kick-down' then it would have been better to move MongoConnection.GetMongoCollection() to Collection.GetMongCollection() as I have added 
Fine with involving Steve on this

Reply:
removed